src/test/java/harness/TestUsers.java

```diff
@@ -41,6 +41,9 @@
   public final String email;
   public final SpendEnabled spendEnabled;
 
+  // name of the group that includes CLI test users and has spend profile access
+  public static final String CLI_TEST_USERS_GROUP_NAME = "cli-test-users";
```

Comment:
Maybe good to say if this was set up once manually? Will the tests using this fail on non dev Sam?

Reply:
Maybe some elaboration on this in the `CONTRIBUTING.md` "Testing" section? (not nesc this PR)

Reply:
RE documentation:
- There's a comment at the top of the class: "These permissions were configured manually, and are not part of the CLI test harness. See CONTRIBUTING.md for more details about the manual setup.".
- `CONTRIBUTING.md` has a [section](https://github.com/DataBiosphere/terra-cli/blob/main/CONTRIBUTING.md#setup-test-users) "Setup test users". I do need to update the command option names there though.

RE tests on non dev SAM:
- Yes, these tests will fail on any SAM instance that hasn't been setup with these test users and group.
- Setting up a new instance should just require re-running the same script. However, if we want to use test users in a different domain (i.e. not `test.firecloud.org`), that would require some refactoring.